app/uid/models.py

```diff
@@ -1,12 +1,29 @@
 from django.db import models
 from neomodel import StructuredNode, StringProperty, DateTimeProperty, BooleanProperty, IntegerProperty, RelationshipTo
 from datetime import datetime
-import uuid
+# from .utils import generate_uid # Import from generate_uid
+# import uuid
+# import hashlib
 
 # Create your models here.
 
+class UIDGenerator:
```

Comment:
Hey, I noticed the class is not attached to any persistence (e.g. models.Model). If the app is restarted, the counter is reset to 0, and UIDs are started from 0 again. I believe we would want to continue wherever the previous UID left off on